.travis.yml

```diff
@@ -1,44 +1,19 @@
-# config file for automatic testing at travis-ci.org
-language: python
 
-python:
-    - "2.7"
+language: python
 
 sudo: false
-env:
-  global:
-    - LD_PRELOAD=/lib/x86_64-linux-gnu/libSegFault.so
-    - SEGFAULT_SIGNALS=all
-  matrix:
-    - TOXENV=check
-#    - TOXENV=docs
 
-    - TOXENV=py27,codecov
-#    - TOXENV=py33,codecov
-#    - TOXENV=py34,codecov
-#    - TOXENV=py35,codecov
-#    - TOXENV=pypy,codecov
-before_install:
-  - python --version
-  - uname -a
-  - lsb_release -a
+python:
+   - "2.7"
+   - "3.5"
+   - "3.6"
+os:
+  - linux
+
 install:
-  - pip install tox
-  - virtualenv --version
-  - easy_install --version
-  - pip --version
-  - tox --version
-script:
-  - tox -v
-after_failure:
-  - more .tox/log/* | cat
-  - more .tox/*/log/* | cat
-before_cache:
-  - rm -rf $HOME/.cache/pip/log
-cache:
-  directories:
-    - $HOME/.cache/pip
-notifications:
-  email:
-    on_success: never
-    on_failure: always
+  - pip install -r requirements.txt
+  - if [[ $TRAVIS_PYTHON_VERSION == 2.7 ]]; then pip install http://download.pytorch.org/whl/cpu/torch-0.3.1-cp27-cp27mu-linux_x86_64.whl; fi
```

Comment:
This line and subsequent assume `Linux`

Reply:
The Travis is only testing now on Linux.

+os:
+  - linux